src/awkward/contents/content.py

```diff
@@ -13,6 +13,7 @@
 
 np = ak._nplikes.NumpyMetadata.instance()
 numpy = ak._nplikes.Numpy.instance()
+NumpyLike = ak._nplikes.NumpyLike
```

Comment:
This looked like a clear bug, so I fixed it instead of ignoring it

Reply:
Fab! Our type hints are 100% human-only for now, but I will start getting mypy properly consuming them soon.